nexus/src/external_api/http_entrypoints.rs

```diff
@@ -2237,6 +2264,416 @@ async fn certificate_delete(
     apictx.external_latencies.instrument_dropshot_handler(&rqctx, handler).await
 }
 
+/// Create an address lot.
+#[endpoint {
+    method = POST,
+    path = "/v1/system/networking/address-lot",
+    tags = ["external-networking"],
```

Comment:
For now we should replace all of these. Probably would be easiest for you to just do a find and replace. We'll improve the tags later, but for now all the system level stuff is grouped together. 

```suggestion
    tags = ["system"],
```

Reply:
Will do!